zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -248,6 +248,7 @@ mod tests {
                 memo BLOB,
                 spent INTEGER,
                 commitment_tree_position INTEGER,
+                recipient_key_scope INTEGER NOT NULL DEFAULT 0,
```

Comment:
* Add a comment that this is External scope.
* Do we actually need this DEFAULT?

Reply:
According to @str4d this is just a side effect of how Sqlite implements the ALTER COLUMN.